server/tinylicious/package.json

```diff
@@ -29,17 +29,18 @@
   },
   "dependencies": {
     "@fluidframework/common-utils": "^0.26.0",
-    "@fluidframework/gitresources": "^0.1017.1",
-    "@fluidframework/protocol-base": "^0.1017.1",
-    "@fluidframework/protocol-definitions": "^0.1017.1",
-    "@fluidframework/server-lambdas": "^0.1017.1",
-    "@fluidframework/server-local-server": "^0.1017.1",
-    "@fluidframework/server-memory-orderer": "^0.1017.1",
-    "@fluidframework/server-services-client": "^0.1017.1",
-    "@fluidframework/server-services-core": "^0.1017.1",
-    "@fluidframework/server-services-shared": "^0.1017.1",
-    "@fluidframework/server-services-utils": "^0.1017.1",
-    "@fluidframework/server-test-utils": "^0.1017.1",
+    "@fluidframework/gitresources": "^0.1025.0-0",
+    "@fluidframework/protocol-base": "^0.1025.0-0",
+    "@fluidframework/protocol-definitions": "^0.1025.0-0",
+    "@fluidframework/server-lambdas": "^0.1025.0-0",
+    "@fluidframework/server-local-server": "^0.1025.0-0",
+    "@fluidframework/server-memory-orderer": "^0.1025.0-0",
+    "@fluidframework/server-routerlicious-base": "^0.1025.0-23979",
```

Comment:
Unfortunate side-effect of #5663 (and its subsequent fix #5801) is that we need to include r11s-base just to get `runService` (similarly in historian). This pulls in all kinds of dependencies like node-rdkafka that are unnecessary here. @curtisman, @GaryWilber, or @tanviraumi do you have any suggestions?

Reply:
Also, it appears this added dependency on node-rdkafka is causing CI to fail because `make` is not included in the pipeline.

Reply:
@GaryWilber tinylicious can't take a dependency on routerlicious-base. Can you recommend us where to put it?

Reply:
Messaged with Gary offline, we decided the best place would be `services-shared`